resources/views/ingame/admin/developershortcuts.blade.php

```diff
@@ -17,52 +17,73 @@
-                        <div class="group bborder" style="display: block;">
-                            <div class="fieldwrapper" style="margin-bottom: 50px;">
-                                <input type="submit" class="btn_blue" name="reset_buildings" value="Set all buildings to level 0">
-                                <input type="submit" class="btn_blue" name="reset_research" value="Set all research to level 0">
-                                <input type="submit" class="btn_blue" name="reset_units" value="Remove all units">
-                            </div>
-                        </div>
-                    </div>
+                                <p class="box_highlight textCenter no_buddies">@lang('Reset planet')</p>
+                                <div class="group bborder" style="display: block;">
+                                    <div class="fieldwrapper" style="margin-bottom: 50px;">
+                                        <input type="submit" class="btn_blue" name="reset_buildings" value="Set all buildings to level 0">
+                                        <input type="submit" class="btn_blue" name="reset_research" value="Set all research to level 0">
+                                        <input type="submit" class="btn_blue" name="reset_units" value="Remove all units">
+                                    </div>
+                                </div>
+                            </form>
 
-                    <div class="footer">
-                    </div>
+                            <form action="{{ route('admin.developershortcuts.update-resources') }}" name="form" method="post">
+                                {{ csrf_field() }}
+                                <p class="box_highlight textCenter no_buddies">@lang('Add / Subtract X of resource to current planet:')</p>
+                                <div class="group bborder" style="display: block;">
+                                    <div class="fieldwrapper">
+                                        <div class="smallFont">You can enter positive or negative values to add or subtract to the selected resource.</div>
+                                        <label class="styled textBeefy">Amount of resources to add / subtract :</label>
+                                        <div class="thefield">
+                                            <input type="text" pattern="^([-+,0-9.]+)" class="textInput w50 textCenter textBeefy" value="1" size="2" name="amount_of_resources">
+                                        </div>
+                                    </div>
+                                    <div class="fieldwrapper" style="margin-bottom: 50px;">
+                                        @foreach (\OGame\Models\Enums\ResourceType::cases() as $resource)
+                                            <input type="submit" name="resource_{{ $resource->value }}" class="btn_blue" value="{{$resource->value}}">
```

Comment:
💡 Small nitpick: could you make the first character of the label of the input button be uppercase? So to make it `Metal` instead of `metal`. I think this looks nicer and conforms to the rest of the game where resources are mentioned.